From 1b824c3835be12bb72421d3ed1e6f53fb1e4bb6a Mon Sep 17 00:00:00 2001
From: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
Date: Tue, 18 Mar 2025 10:27:54 +0100
Subject: [PATCH 1618/1695] dts: stm32: add st,ck_gpu node for stm32mp25 boards

st,ck_gpu not is used to define GPU OPPs.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 9d1610da95ba0e47f7da4a5f5bf0b6fe25db0ef1 dts: stm32: add st,ck_gpu node for stm32mp25 boards]
Signed-off-by: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
Change-Id: Ie43bfc2c199a6b8b1662706c716724a6dd508deb
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/444789
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 core/arch/arm/dts/stm32mp235f-dk.dts  |  6 ++++++
 core/arch/arm/dts/stm32mp257f-dk.dts  | 11 +++++++++++
 core/arch/arm/dts/stm32mp257f-ev1.dts | 11 +++++++++++
 3 files changed, 28 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp235f-dk.dts b/core/arch/arm/dts/stm32mp235f-dk.dts
index 27877de67..49a4b0b08 100644
--- a/core/arch/arm/dts/stm32mp235f-dk.dts
+++ b/core/arch/arm/dts/stm32mp235f-dk.dts
@@ -397,6 +397,12 @@
 				st,pll = <&pll1_cfg_1200Mhz>;
 			};
 		};
+		st,ck_gpu {
+			cfg_1 {
+				hz = <400000000>;
+				st,pll = <&pll3_cfg_400Mhz>;
+			};
+		};
 	};
 };
 
diff --git a/core/arch/arm/dts/stm32mp257f-dk.dts b/core/arch/arm/dts/stm32mp257f-dk.dts
index f48943808..ec175bb67 100644
--- a/core/arch/arm/dts/stm32mp257f-dk.dts
+++ b/core/arch/arm/dts/stm32mp257f-dk.dts
@@ -397,6 +397,17 @@
 				st,pll = <&pll1_cfg_1200Mhz>;
 			};
 		};
+		st,ck_gpu {
+			cfg_1 {
+				hz = <800000000>;
+				st,pll = <&pll3_cfg_800Mhz>;
+			};
+
+			cfg_2 {
+				hz = <900000000>;
+				st,pll = <&pll3_cfg_900Mhz>;
+			};
+		};
 	};
 };
 
diff --git a/core/arch/arm/dts/stm32mp257f-ev1.dts b/core/arch/arm/dts/stm32mp257f-ev1.dts
index b24694527..053037ea7 100644
--- a/core/arch/arm/dts/stm32mp257f-ev1.dts
+++ b/core/arch/arm/dts/stm32mp257f-ev1.dts
@@ -439,6 +439,17 @@
 				st,pll = <&pll1_cfg_1200Mhz>;
 			};
 		};
+		st,ck_gpu {
+			cfg_1 {
+				hz = <800000000>;
+				st,pll = <&pll3_cfg_800Mhz>;
+			};
+
+			cfg_2 {
+				hz = <900000000>;
+				st,pll = <&pll3_cfg_900Mhz>;
+			};
+		};
 	};
 };
 
-- 
2.47.2

